### ISSUE TITLE
Multiple checkboxes have gone uncheked after scroll with gridview: true

### ISSUE THREAD

**Issue by AuthorProxy** [tonytomov/jqGrid#691]
Example from http://www.trirand.net/demo/javascript/jqgrid/loading_data/scrollbar/index.html
Versions of Guriddo jqGrid JS: 4.7.0 and 4.7.1. Add `gridview: true` to repeat bug

```
(function ($) {
  'use strict';
  $(function () {
    $('#jqGrid').jqGrid({
      url: 'http://trirand.com/blog/phpjqgrid/examples/jsonp/getjsonp.php?callback=?&qwery=longorders',
      mtype: 'GET',
      datatype: 'json',
      page: 1,
      colNames: ['Order ID', 'Customer ID', 'Order Date', 'Freight', 'Ship Name'],
      colModel: [{
        name: 'OrderID',
        key: true,
        width: 75
      }, {
        name: 'CustomerID',
        width: 150
      }, {
        name: 'OrderDate',
        width: 150
      }, {
        name: 'Freight',
        width: 150
      }, {
        name: 'ShipName',
        width: 150
      }],
      width: 750,
      height: 250,
      rowNum: 20,
      gridview: true,
      scroll: 1, // set the scroll property to 1 to enable paging with scrollbar - virtual loading of records
      emptyrecords: 'Scroll to bottom to retrieve new page', // the message will be displayed at the bottom
      pager: '#jqGridPager'
    });
  });
})(jQuery);
```

When you select some checkboxes on the page `X` and scroll down, and then go back to page `X`, checkboxes become unchecked, but selection is still applied, and `.jqGrid('getGridParam', 'selarrrow')` returns selected rows. The problem repeats only with `gridview: true`. I make some [demo screencast](http://screencast.com/t/EgAaMNsLOWp) that repeat bug.

**Comment by tonytomov** [comment]
Hello,

I can not reproduce this with the last development version. I can make a demo for this purpose

**Comment by AuthorProxy** [comment]
Fixed by disabling gridview (set to false)

**Comment by tonytomov** [comment]
Hello,

Ok. I will chsck this and will replay to this soon.

**Comment by tonytomov** [comment]
Hello,

This demo show the multiselect with scroll :1 and gridview set to true example which, works as expected.
http://www.guriddo.net/demo/guriddojs/loading_data/scrollmulti/

Please let me know if you have problem with this.

Kind Regards

**Comment by AuthorProxy** [comment]
Yes I have problems on your demo, it doens't work as expected cause selected rows are still selected, but not checked http://screencast.com/t/hbXICAZZgbsb.

**Comment by OlegKi** [comment]
I want to add that I fixed the same problem in free jqGrid by usage [the fix](https://github.com/free-jqgrid/jqGrid/commit/1e6625e4a6284f409f29b65ab9f9309c9846ac12#diff-2d010088e9cda4a876dbc3e2399c83a0R2675), which I posted in [my answer](http://stackoverflow.com/a/29142519/315935) on the stackoverlow which you referenced. The reason was just the state of the checkboxs was set by changing of *properties* instead of *attributes*. Thus the existing code worked incorrectly.

**Comment by tonytomov** [comment]
@AuthorProxy 
Thanks. Firsty of all I was unable to understand excellent your problem. I was able to reproduce the bug.

Thanks

**Comment by tonytomov** [comment]
Hello,

I have updated the demo http://www.guriddo.net/demo/guriddojs/loading_data/scrollmulti/
according to this fix:  https://github.com/tonytomov/jqGrid/commit/1623d836aee8437739ab75c22be969ecbebec257

Could you please confirm that the problem is solved?
Thank you

**Comment by AuthorProxy** [comment]
@tonytomov @OlegKi  Almost fixed, but the top checkbox (check all) is still checked always, even if no row is selected.
1. Check all (top cb checked, rows checked)
2. Scroll to the center or end of list (top cb checked, rows unchecked)
3. Back to the top (top cb checked, rows unchecked)

**Comment by tonytomov** [comment]
Hello,

This is true and will be fixed in the next releases

**Comment by SachinThorkar** [comment]
try it by  adding " multiselect : true " , it might give solution
